from PySide6.QtCore import (QCoreApplication, QLocale,
    QMetaObject, QRect,
    Qt)
from PySide6.QtGui import (QAction, QBrush, QColor, QPalette, QPixmap)
from PySide6.QtWidgets import (QFrame, QLabel, QPushButton, QStatusBar, QWidget)

class Ui_Main(object):
    def setupUi(self, Main):
        if not Main.objectName():
            Main.setObjectName(u"Main")
        Main.resize(1920, 1080)
        palette = QPalette()
        brush = QBrush(QColor(255, 227, 203, 255))
        brush.setStyle(Qt.SolidPattern)
        palette.setBrush(QPalette.Active, QPalette.WindowText, brush)
        brush1 = QBrush(QColor(26, 110, 175, 255))
        brush1.setStyle(Qt.SolidPattern)
        palette.setBrush(QPalette.Active, QPalette.Button, brush1)
        brush2 = QBrush(QColor(136, 136, 136, 255))
        brush2.setStyle(Qt.SolidPattern)
        palette.setBrush(QPalette.Active, QPalette.Light, brush2)
        palette.setBrush(QPalette.Active, QPalette.Text, brush)
        palette.setBrush(QPalette.Active, QPalette.ButtonText, brush)
        brush3 = QBrush(QColor(0, 0, 0, 255))
        brush3.setStyle(Qt.SolidPattern)
        palette.setBrush(QPalette.Active, QPalette.Base, brush3)
        brush4 = QBrush(QColor(120, 131, 170, 255))
        brush4.setStyle(Qt.SolidPattern)
        palette.setBrush(QPalette.Active, QPalette.Window, brush4)
        palette.setBrush(QPalette.Active, QPalette.Shadow, brush)
        palette.setBrush(QPalette.Active, QPalette.HighlightedText, brush)
        brush5 = QBrush(QColor(255, 255, 255, 255))
        brush5.setStyle(Qt.SolidPattern)
        palette.setBrush(QPalette.Active, QPalette.ToolTipBase, brush5)
        palette.setBrush(QPalette.Active, QPalette.ToolTipText, brush5)
        brush6 = QBrush(QColor(85, 0, 255, 128))
        brush6.setStyle(Qt.SolidPattern)
#if QT_VERSION >= QT_VERSION_CHECK(5, 12, 0)
        palette.setBrush(QPalette.Active, QPalette.PlaceholderText, brush6)
#endif
        palette.setBrush(QPalette.Inactive, QPalette.WindowText, brush)
        palette.setBrush(QPalette.Inactive, QPalette.Button, brush1)
        brush7 = QBrush(QColor(255, 85, 255, 255))
        brush7.setStyle(Qt.SolidPattern)
        palette.setBrush(QPalette.Inactive, QPalette.Window, brush7)
        palette.setBrush(QPalette.Inactive, QPalette.HighlightedText, brush)
        palette.setBrush(QPalette.Inactive, QPalette.ToolTipBase, brush5)
        palette.setBrush(QPalette.Disabled, QPalette.Text, brush5)
        palette.setBrush(QPalette.Disabled, QPalette.ButtonText, brush5)
        palette.setBrush(QPalette.Disabled, QPalette.HighlightedText, brush)
        Main.setPalette(palette)
        self.actionAdd_Image = QAction(Main)
        self.actionAdd_Image.setObjectName(u"actionAdd_Image")
        self.actionAdd_Image.setMenuRole(QAction.MenuRole.ApplicationSpecificRole)
        self.centralwidget = QWidget(Main)
        self.centralwidget.setObjectName(u"centralwidget")
        self.frame = QFrame(self.centralwidget)
        self.frame.setObjectName(u"frame")
        self.frame.setGeometry(QRect(-6, 198, 1941, 786))
        self.frame.setFrameShape(QFrame.Shape.StyledPanel)
        self.frame.setFrameShadow(QFrame.Shadow.Raised)
        self.ncbutton = QPushButton(self.frame)
        self.ncbutton.setObjectName(u"ncbutton")
        self.ncbutton.setGeometry(QRect(553, 121, 121, 41))
        self.ncbutton.setMouseTracking(False)
        self.ncbutton.setContextMenuPolicy(Qt.ContextMenuPolicy.DefaultContextMenu)
        self.ncbutton.setAcceptDrops(False)
        self.ncbutton.setAutoRepeatDelay(301)
        self.ttcbutton = QPushButton(self.frame)
        self.ttcbutton.setObjectName(u"ttcbutton")
        self.ttcbutton.setGeometry(QRect(743, 121, 121, 41))
        self.garmpricingbutton = QPushButton(self.frame)
        self.garmpricingbutton.setObjectName(u"garmpricingbutton")
        self.garmpricingbutton.setGeometry(QRect(943, 121, 121, 41))
        self.gandccbutton = QPushButton(self.frame)
        self.gandccbutton.setObjectName(u"gandccbutton")
        self.gandccbutton.setGeometry(QRect(1133, 121, 151, 41))
        self.tocbutton = QPushButton(self.frame)
        self.tocbutton.setObjectName(u"tocbutton")
        self.tocbutton.setGeometry(QRect(1343, 121, 131, 41))
        self.createempbutton = QPushButton(self.frame)
        self.createempbutton.setObjectName(u"createempbutton")
        self.createempbutton.setGeometry(QRect(540, 271, 130, 49))
        self.managedeliveriesbutton = QPushButton(self.frame)
        self.managedeliveriesbutton.setObjectName(u"managedeliveriesbutton")
        self.managedeliveriesbutton.setGeometry(QRect(950, 268, 143, 43))
        self.reportsbutton = QPushButton(self.frame)
        self.reportsbutton.setObjectName(u"reportsbutton")
        self.reportsbutton.setGeometry(QRect(1159, 265, 142, 41))
        self.billingbutton = QPushButton(self.frame)
        self.billingbutton.setObjectName(u"billingbutton")
        self.billingbutton.setGeometry(QRect(749, 271, 137, 46))
        self.emailcustbutton = QPushButton(self.frame)
        self.emailcustbutton.setObjectName(u"emailcustbutton")
        self.emailcustbutton.setGeometry(QRect(1346, 263, 146, 43))
        self.bannerframe = QFrame(self.centralwidget)
        self.bannerframe.setObjectName(u"bannerframe")
        self.bannerframe.setGeometry(QRect(-32, 8, 1932, 190))
        palette1 = QPalette()
        palette1.setBrush(QPalette.Active, QPalette.WindowText, brush3)
        brush8 = QBrush(QColor(56, 139, 175, 255))
        brush8.setStyle(Qt.SolidPattern)
        palette1.setBrush(QPalette.Active, QPalette.Button, brush8)
        brush9 = QBrush(QColor(89, 205, 255, 255))
        brush9.setStyle(Qt.SolidPattern)
        palette1.setBrush(QPalette.Active, QPalette.Light, brush9)
        brush10 = QBrush(QColor(72, 172, 215, 255))
        brush10.setStyle(Qt.SolidPattern)
        palette1.setBrush(QPalette.Active, QPalette.Midlight, brush10)
        brush11 = QBrush(QColor(28, 69, 87, 255))
        brush11.setStyle(Qt.SolidPattern)
        palette1.setBrush(QPalette.Active, QPalette.Dark, brush11)
        brush12 = QBrush(QColor(37, 93, 117, 255))
        brush12.setStyle(Qt.SolidPattern)
        palette1.setBrush(QPalette.Active, QPalette.Mid, brush12)
        palette1.setBrush(QPalette.Active, QPalette.Text, brush3)
        palette1.setBrush(QPalette.Active, QPalette.BrightText, brush5)
        palette1.setBrush(QPalette.Active, QPalette.ButtonText, brush3)
        brush13 = QBrush(QColor(65, 65, 65, 255))
        brush13.setStyle(Qt.SolidPattern)
        palette1.setBrush(QPalette.Active, QPalette.Base, brush13)
        palette1.setBrush(QPalette.Active, QPalette.Window, brush8)
        palette1.setBrush(QPalette.Active, QPalette.Shadow, brush3)
        brush14 = QBrush(QColor(155, 197, 215, 255))
        brush14.setStyle(Qt.SolidPattern)
        palette1.setBrush(QPalette.Active, QPalette.AlternateBase, brush14)
        brush15 = QBrush(QColor(255, 255, 220, 255))
        brush15.setStyle(Qt.SolidPattern)
        palette1.setBrush(QPalette.Active, QPalette.ToolTipBase, brush15)
        palette1.setBrush(QPalette.Active, QPalette.ToolTipText, brush3)
        brush16 = QBrush(QColor(0, 0, 0, 127))
        brush16.setStyle(Qt.SolidPattern)
#if QT_VERSION >= QT_VERSION_CHECK(5, 12, 0)
        palette1.setBrush(QPalette.Active, QPalette.PlaceholderText, brush16)
#endif
        palette1.setBrush(QPalette.Active, QPalette.Accent, brush5)
        palette1.setBrush(QPalette.Inactive, QPalette.WindowText, brush3)
        palette1.setBrush(QPalette.Inactive, QPalette.Button, brush8)
        palette1.setBrush(QPalette.Inactive, QPalette.Light, brush9)
        palette1.setBrush(QPalette.Inactive, QPalette.Midlight, brush10)
        palette1.setBrush(QPalette.Inactive, QPalette.Dark, brush11)
        palette1.setBrush(QPalette.Inactive, QPalette.Mid, brush12)
        palette1.setBrush(QPalette.Inactive, QPalette.Text, brush3)
        palette1.setBrush(QPalette.Inactive, QPalette.BrightText, brush5)
        palette1.setBrush(QPalette.Inactive, QPalette.ButtonText, brush3)
        palette1.setBrush(QPalette.Inactive, QPalette.Base, brush13)
        palette1.setBrush(QPalette.Inactive, QPalette.Window, brush8)
        palette1.setBrush(QPalette.Inactive, QPalette.Shadow, brush3)
        palette1.setBrush(QPalette.Inactive, QPalette.AlternateBase, brush14)
        palette1.setBrush(QPalette.Inactive, QPalette.ToolTipBase, brush15)
        palette1.setBrush(QPalette.Inactive, QPalette.ToolTipText, brush3)
#if QT_VERSION >= QT_VERSION_CHECK(5, 12, 0)
        palette1.setBrush(QPalette.Inactive, QPalette.PlaceholderText, brush16)
#endif
        palette1.setBrush(QPalette.Inactive, QPalette.Accent, brush5)
        palette1.setBrush(QPalette.Disabled, QPalette.WindowText, brush11)
        palette1.setBrush(QPalette.Disabled, QPalette.Button, brush8)
        palette1.setBrush(QPalette.Disabled, QPalette.Light, brush9)
        palette1.setBrush(QPalette.Disabled, QPalette.Midlight, brush10)
        palette1.setBrush(QPalette.Disabled, QPalette.Dark, brush11)
        palette1.setBrush(QPalette.Disabled, QPalette.Mid, brush12)
        palette1.setBrush(QPalette.Disabled, QPalette.Text, brush11)
        palette1.setBrush(QPalette.Disabled, QPalette.BrightText, brush5)
        palette1.setBrush(QPalette.Disabled, QPalette.ButtonText, brush11)
        palette1.setBrush(QPalette.Disabled, QPalette.Base, brush8)
        palette1.setBrush(QPalette.Disabled, QPalette.Window, brush8)
        palette1.setBrush(QPalette.Disabled, QPalette.Shadow, brush3)
        palette1.setBrush(QPalette.Disabled, QPalette.AlternateBase, brush8)
        palette1.setBrush(QPalette.Disabled, QPalette.ToolTipBase, brush15)
        palette1.setBrush(QPalette.Disabled, QPalette.ToolTipText, brush3)
        brush17 = QBrush(QColor(28, 69, 87, 127))
        brush17.setStyle(Qt.SolidPattern)
#if QT_VERSION >= QT_VERSION_CHECK(5, 12, 0)
        palette1.setBrush(QPalette.Disabled, QPalette.PlaceholderText, brush17)
#endif
        brush18 = QBrush(QColor(73, 181, 227, 255))
        brush18.setStyle(Qt.SolidPattern)
        palette1.setBrush(QPalette.Disabled, QPalette.Accent, brush18)
        self.bannerframe.setPalette(palette1)
        self.bannerframe.setAutoFillBackground(True)
        self.bannerframe.setFrameShape(QFrame.Shape.StyledPanel)
        self.bannerframe.setFrameShadow(QFrame.Shadow.Raised)
        self.chooseimage = QLabel(self.bannerframe)
        self.chooseimage.setObjectName(u"chooseimage")
        self.chooseimage.setGeometry(QRect(719, 2, 566, 184))
        self.chooseimage.setFocusPolicy(Qt.FocusPolicy.WheelFocus)
        self.chooseimage.setToolTipDuration(-2)
        self.chooseimage.setLocale(QLocale(QLocale.Ewe, QLocale.Ghana))
        self.chooseimage.setPixmap(QPixmap(u":/images/Logo2.jpg"))
        self.chooseimage.setScaledContents(True)
        self.hanger1 = QLabel(self.bannerframe)
        self.hanger1.setObjectName(u"hanger1")
        self.hanger1.setGeometry(QRect(31, 63, 190, 183))
        self.hanger1.setAutoFillBackground(False)
        self.hanger1.setPixmap(QPixmap(u"../PyFiles2/resources/hanger3.png"))
        self.hanger1.setScaledContents(True)
        self.hanger2 = QLabel(self.bannerframe)
        self.hanger2.setObjectName(u"hanger2")
        self.hanger2.setGeometry(QRect(127, 57, 201, 187))
        self.hanger2.setAutoFillBackground(False)
        self.hanger2.setPixmap(QPixmap(u"../PyFiles2/resources/hanger4.png"))
        self.hanger2.setScaledContents(True)
        self.hanger3 = QLabel(self.bannerframe)
        self.hanger3.setObjectName(u"hanger3")
        self.hanger3.setGeometry(QRect(308, -2, 201, 187))
        self.hanger3.setAutoFillBackground(False)
        self.hanger3.setPixmap(QPixmap(u"../PyFiles2/resources/hanger1.png"))
        self.hanger3.setScaledContents(True)
        self.hanger4 = QLabel(self.bannerframe)
        self.hanger4.setObjectName(u"hanger4")
        self.hanger4.setGeometry(QRect(523, 4, 186, 184))
        self.hanger4.setAutoFillBackground(False)
        self.hanger4.setPixmap(QPixmap(u"../PyFiles2/resources/hanger2.png"))
        self.hanger4.setScaledContents(True)
        self.hanger8 = QLabel(self.bannerframe)
        self.hanger8.setObjectName(u"hanger8")
        self.hanger8.setGeometry(QRect(1809, 5, 186, 184))
        self.hanger8.setAutoFillBackground(False)
        self.hanger8.setPixmap(QPixmap(u"../PyFiles2/resources/hanger2.png"))
        self.hanger8.setScaledContents(True)
        self.hanger6 = QLabel(self.bannerframe)
        self.hanger6.setObjectName(u"hanger6")
        self.hanger6.setGeometry(QRect(1407, 62, 201, 187))
        self.hanger6.setAutoFillBackground(False)
        self.hanger6.setPixmap(QPixmap(u"../PyFiles2/resources/hanger4.png"))
        self.hanger6.setScaledContents(True)
        self.hanger5 = QLabel(self.bannerframe)
        self.hanger5.setObjectName(u"hanger5")
        self.hanger5.setGeometry(QRect(1305, 66, 190, 183))
        self.hanger5.setAutoFillBackground(False)
        self.hanger5.setPixmap(QPixmap(u"../PyFiles2/resources/hanger3.png"))
        self.hanger5.setScaledContents(True)
        self.hanger7 = QLabel(self.bannerframe)
        self.hanger7.setObjectName(u"hanger7")
        self.hanger7.setGeometry(QRect(1576, 4, 201, 187))
        self.hanger7.setAutoFillBackground(False)
        self.hanger7.setPixmap(QPixmap(u"../PyFiles2/resources/hanger1.png"))
        self.hanger7.setScaledContents(True)
        Main.setCentralWidget(self.centralwidget)
        self.statusbar = QStatusBar(Main)
        self.statusbar.setObjectName(u"statusbar")
        Main.setStatusBar(self.statusbar)

        self.retranslateUi(Main)

        self.ncbutton.setDefault(False)


        QMetaObject.connectSlotsByName(Main)
    # setupUi

    def retranslateUi(self, Main):
        Main.setWindowTitle(QCoreApplication.translate("Main", u"MainWindow", None))
        self.actionAdd_Image.setText(QCoreApplication.translate("Main", u"Add Image", None))
#if QT_CONFIG(shortcut)
        self.actionAdd_Image.setShortcut(QCoreApplication.translate("Main", u"Ctrl+Alt+I", None))
#endif // QT_CONFIG(shortcut)
        self.ncbutton.setText(QCoreApplication.translate("Main", u"Next Customer", None))
        self.ttcbutton.setText(QCoreApplication.translate("Main", u"Ticket Type Creation", None))
        self.garmpricingbutton.setText(QCoreApplication.translate("Main", u"Garment Pricing", None))
        self.gandccbutton.setText(QCoreApplication.translate("Main", u"Garment + Color Creation", None))
        self.tocbutton.setText(QCoreApplication.translate("Main", u"Ticket Options Creation", None))
        self.createempbutton.setText(QCoreApplication.translate("Main", u"Create/Edit Employees", None))
        self.managedeliveriesbutton.setText(QCoreApplication.translate("Main", u"Manage Deliveries", None))
        self.reportsbutton.setText(QCoreApplication.translate("Main", u"Reports", None))
        self.billingbutton.setText(QCoreApplication.translate("Main", u"Billing", None))
        self.emailcustbutton.setText(QCoreApplication.translate("Main", u"Email Customers", None))
        self.chooseimage.setText("")
        self.hanger1.setText("")
        self.hanger2.setText("")
        self.hanger3.setText("")
        self.hanger4.setText("")
        self.hanger8.setText("")
        self.hanger6.setText("")
        self.hanger5.setText("")
        self.hanger7.setText("")